src/reversion_rest_framework/mixins.py

```diff
@@ -15,11 +16,24 @@ class BaseHistoryModelMixin:
 class HistoryOnlyMixin(BaseHistoryModelMixin):
     @action(detail=True, methods=['GET'], name='Get History')
     def history(self, request, pk=None):
-        object = self.get_object()
-        versions = Version.objects.get_for_object(object)
+        instance = self.get_object()
+        versions = Version.objects.get_for_object(instance).order_by(
+            "revision__created_date"
+        )
+        page = self.paginate_queryset(versions)
+        if page is not None:
+            serializer = self.version_serializer(page, many=True)
+            return self.get_paginated_response(serializer.data)
         serializer = self.version_serializer(versions, many=True)
         return Response(serializer.data)
 
+    @action(detail=True, methods=['GET'], name='Get Historic Version', url_path=r"history/(?P<version_pk>\d+)")
```

Comment:
the `url_path` is required to let the router know:
1. that the route is shared with `history`
2. that there is an additional`version_pk`

not pleasant I know. better ideas welcome

Reply:
Well, this sounds great 😉